AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Commands/AWSSDKSwiftCLI/Subcommands/GeneratePackageVersion.swift

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import ArgumentParser
+import Foundation
+import AWSCLIUtils
+
+struct GeneratePackageVersionCommand: ParsableCommand {
```

Comment:
Just a simple Swift CLI command that calls the package version builder (code below).